aframe/index.html

```diff
@@ -27,8 +27,7 @@
       <a-box color="tomato" depth="4" height="4" width="0.5" position="2.5 2 2"></a-box>
 
       <!-- Our ads -->
-      <a-entity visible="true" zesty-ad="auId: ce6f68fc-4809-4409-8f57-c631283ce5a3" position="0 2 0"></a-entity>
-      <a-zesty-ad visible="true" position="4 2 0"></a-zesty-ad>
+      <a-entity visible="true" zesty-ad="tokenGroup: 0; publisher: 0xC22F22d096D578148FFDbe969E89b0fe5Cb9d5aB" position="0 2 0"></a-entity>
```

Comment:
Would be very handy if we could auto-generate this for the publisher from their account/NFT data.